R/plot_diagram.R

```diff
@@ -30,58 +32,178 @@ plot_diagram <- function(data = NULL) {
   data_equity <- data %>%
     filter(.data$asset_class == "equity")
 
-  DiagrammeR::grViz(glue("digraph {
-  graph [rankdir = LR]
-
-  node [shape = rectangle]
-
-  subgraph cluster_total_exposure {
-    peripheries = 0
-
-    label = 'Total Portfolio Exposure'
-
-    total_exp [label = '{{formatC(data_equity$exposure_portfolio[1], format='f', big.mark=',', digits=0)}} CHF']
-  }
-
-  subgraph cluster_asset_class_exposure {
-    peripheries = 0
+  currency_id <- "CHF"
+  emissions_unit <- "tonnes"
 
-    label = 'Exposure per asset class'
+  ttl_exp_lbl <- paste0(scales::label_comma()(data_equity$exposure_portfolio), " CHF")
```

Comment:
Seems like currency should be a parameter?

Reply:
yes, but I left decisions like that (like changing the function signature) to @MonikaFu

Reply:
crap, you're right... I forgot to add the parameter here

Reply:
fixed

Reply:
totally fair